### PR TITLE
feat: Theoros validators stuff

### PR DESCRIPTION
**Pull request by akhercha**
Resolves: #NA

## Work done
* `data_feeds` endpoint base,
* `TheorosStorage`,
* indexing of announced locations,
* `PragmaWrapperCalls` rpc stuff
* pretty json for built docs
* removal of ̀`.env` in prepartion of the future configuration stuff

### CODE REVIEW

File: rust/theoros/src/rpc/mod.rs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+impl StarknetRpc {
+    pub fn new(rpc_url: Url) -> Self {
+        Self(JsonRpcClient::new(HttpTransport::new(rpc_url)))
+    }
+}
+
+#[async_trait::async_trait]
+impl HyperlaneCalls for StarknetRpc {
+    async fn get_announced_storage_locations(
```
Comment (EvolveArt) [inline rust/theoros/src/rpc/mod.rs:25]: we should probably use the rust bindings defined [here](https://github.com/astraly-labs/hyperlane_starknet/blob/main/rust/build.rs) for safety

File: rust/theoros/src/types/hyperlane/events/validator_announcement_event.rs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    pub storage_location: String,
+}
+
+impl FromStarknetEventData for ValidatorAnnouncementEvent {
+    fn from_starknet_event_data(mut data: impl Iterator<Item = FieldElement>) -> anyhow::Result<Self> {
+        let validator_as_felt = apibara_field_as_felt(&data.next().context("Missing validator")?);
+        let validator = EthAddress::from_felt(&validator_as_felt).context("Invalid validator ETH address")?;
+
+        let storage_location_felts: Vec<Felt> = data.map(|f| apibara_field_as_felt(&f)).collect();
+        let storage_location = storage_location_felts.to_string();
```
Comment (EvolveArt) [inline rust/theoros/src/types/hyperlane/events/validator_announcement_event.rs:22]: this won't probably work, you need to use `use starknet::core::utils::parse_cairo_short_string` and not just convert to felt

Comment (akhercha) [reply rust/theoros/src/types/hyperlane/events/validator_announcement_event.rs:22]: ah yeah it's misleading but it's actually parsing the felt, it's using my trait `FeltVecToString`... I should probably remove it

File: rust/theoros/src/rpc/mod.rs (hunk 1 above)
Comment (akhercha) [reply rust/theoros/src/rpc/mod.rs:25]: Yup I got an issue for that - will do it in a follow-up 🙏

File: rust/theoros/src/types/hyperlane/events/validator_announcement_event.rs (hunk 2 above)
Comment (EvolveArt) [reply rust/theoros/src/types/hyperlane/events/validator_announcement_event.rs:22]: ah yes indeed it's kind of misleading

**Review by EvolveArt — APPROVED** [review]
LGTM

File: rust/theoros/src/types/hyperlane/events/validator_announcement_event.rs (hunk 2 above)
Comment (akhercha) [reply rust/theoros/src/types/hyperlane/events/validator_announcement_event.rs:22]: Gonna kill it in the next PR 🤝

**Pull request merged**